doc/classes/InputEventKey.xml

```diff
@@ -4,6 +4,7 @@
 		Input event type for keyboard events.
 	</brief_description>
 	<description>
+		Contains keyboard events informations. See [method Node._input] and http://docs.godotengine.org/en/stable/learning/features/inputs/inputevent.html?highlight=input#inputevent.
```

Comment:
Stores key presses on the keyboard. Supports key presses, key releases and [member echo] events.